lib/msgpax.ex

```diff
@@ -27,6 +27,17 @@ defmodule Msgpax do
 
   """
 
+  @type pack_error_reason ::
+    {:too_big, any} |
+    {:not_encodable, any}
+
+  @type unpack_error_reason ::
+    {:extra_bytes, binary} |
```

Comment:
I see your point but I think the word "surplus" is a bit weird and a bit
fancy for programming-related terms. There are synonyms (
http://www.thesaurus.com/browse/surplus) like "excess" that we could use,
wdyt? (btw the definition of "surplus" is "extra" so not sure we would add
anything anyways :/).

On Tuesday, 16 August 2016, Aleksei Magusev <notifications@github.com>
wrote:

> In lib/msgpax.ex
> <https://github.com/lexmag/msgpax/pull/23#discussion_r74849251>:
>
> > @@ -27,6 +27,17 @@ defmodule Msgpax do
> >
> >    """
> >
> > +  @type pack_error_reason ::
> > +    {:too_big, any} |
> > +    {:not_encodable, any}
> > +
> > +  @type unpack_error_reason ::
> > +    {:extra_bytes, binary} |
>
> Let's use surplus_bytes.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/lexmag/msgpax/pull/23/files/2f1beaa3aca28ea8fbaa9319a838779ee18066b6#r74849251>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADtcSkHax59xLsbCPlyP-LKlMx977rrwks5qgOvIgaJpZM4I_9hv>
> .
>


-- 

Andrea Leopardi
an.leopardi@gmail.com